telemetry/specs/INT.mdk

```diff
@@ -1610,6 +1625,11 @@ control EgressDeparserImpl(packet_out packet,
             ck.add({hdr.int_egress_port_tx_util.egress_port_tx_util});
         }
 
+        if (hdr.int_b_occupancy.isValid()) {
+            ck.add({
+                    hdr.int_b_occupancy.b_occupancy
```

Comment:
Same here. If we want buffer_id in line 866, let's make the code consistent.

Reply:
Done.